ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -173,6 +173,17 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-aws-arm64-single-node
+  interval: 48h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
```

Comment:
Whenever masters are scheduleable, pretty sure you need to give them more resources (since they're doing the jobs of both master and worker).  Probably need at least `m6g.2xlarge` here; what does x86 do for this test?

Reply:
They are using the default `COMPUTE_NODE_TYPE` which is set to `m6i, m5` https://github.com/openshift/installer/blob/master/pkg/types/aws/defaults/platform.go#L44

Reply:
I don't know where the default for the second part is set (i.e. `large` or `xlarge`), but the SNO install docs (https://docs.openshift.com/container-platform/4.9/installing/installing_sno/install-sno-preparing-to-install-sno.html) state a machine with at least 8 cores and 33gb RAM is needed, which would map to `m6g.2xlarge`.